File: sangsoo/0x03 array/2577.c
#include <stdio.h>
#include <string.h>
int main(){
    int input, sum = 1;
    int digit;
    int num[10];

    for(int i = 0; i < 10; i++) num[i] = 0;
    for(int i =0; i< 3; i++){
        scanf("%d", &input);
        sum *= input;
    }
    for(int i = 0; sum > 0; i++){
        digit = sum %10;
        num[digit] += 1;
        sum /= 10;
    }
    for(int i = 0; i < 10; i++) printf("%d\n", num[i]);
}

File: sangsoo/0x03 array/2577.py
A = input()
B = input()
C = input()
N = int(A)*int(B)*int(C)
N = str(N)
numdict = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6:0, 7:0, 8:0, 9:0}
for i in range(len(N)):
    numdict[int(N[i])] += 1

values = list(numdict.values())
for i in range(len(values)):
    print(values[i])